server.go

```diff
@@ -118,3 +115,23 @@ func (srv *server) CloseSession(sessionKey string) int {
 	}
 	return len(connections)
 }
+
```

Comment:
Considering my previous comment on desync of `srv.stopping` and `srv.currentOps` I'd, again, move those two fields into a separate structure proctected by a `sync.RWMutex`.